src/containers/CouponsPage/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from "react";
+import { Typography } from "antd";
+import { useTranslation } from "react-i18next";
+import CouponsPageHeader from "../../components/CouponsPageHeader";
+import Coupon from "../../components/Coupon";
+import firebase from "../../firebaseConfig";
+
+const db = firebase.firestore();
+const { Title } = Typography;
+
+const CouponsPage = () => {
+  const { t } = useTranslation();
+  const [availableCoupons, setAvailableCoupons] = useState([]);
+
+  useEffect(() => {
+    db.collection("coupons")
+      .get()
+      .then((querySnapshot) => {
+        const couponsArray = [];
+        querySnapshot.forEach((doc) => couponsArray.push({ ...doc.data() }));
```

Comment:
can i see the kind of data you have here like a sample,

Reply:
![Screenshot from 2020-11-19 00-27-38](https://user-images.githubusercontent.com/67783666/99593921-8041da80-2a03-11eb-96f5-a11ba5adec63.png)